app/controllers/facility_notifications_controller.rb

```diff
@@ -107,7 +107,7 @@ def mark_as_reviewed
   def notify_accounts
     @accounts_to_notify.each do |account, facility|
       account.notify_users.each do |u|
-        Notifier.review_orders(:user => u, :facility => facility, :account => account).deliver
+        Notifier.delay.review_orders(:user => u, :facility => facility, :account => account)
```

Comment:
Have you run into any trouble with deserialization? I remember it being a problem at one point with `AccountTransactionReportMailer.csv_report_email`, and we worked around that by passing in just IDs. But that was also an older `delayed_job` gem version…